demisto_sdk/commands/common/git_tools.py

```diff
@@ -19,7 +19,8 @@ def get_current_working_branch() -> str:
 
 
 def get_changed_files(from_branch: str = 'master', filter_results: Callable = None):
-    temp_files = run_command(f'git diff --name-status {from_branch}').split('\n')
+    temp_files = run_command(
```

Comment:
why did you change all those lines?

Reply:
It’s the auto pep8 precondition configuration, it spit automatically